Duplicati/Library/Utility/SslCertificateValidator.cs

```diff
@@ -31,10 +31,16 @@ public class SslCertificateValidator
         [Serializable]
         public class InvalidCertificateException : Exception
         {
-            private readonly string m_certificate = null;
+            private readonly string? m_certificate = null;
             private readonly SslPolicyErrors m_errors = SslPolicyErrors.None;
 
-            public string Certificate { get { return m_certificate; } }
+            public string Certificate 
+            {
+                get
+                {
+                    return m_certificate ?? throw new InvalidOperationException("Certificate is null");

```

Comment:
There is a check for this property in `RemoteOperation.cs`, I think this property should allow null to be returned.